packages/cli/src/options/beaconNodeOptions/network.ts

```diff
@@ -113,6 +170,27 @@ export const options: CliCommandOptions<NetworkArgs> = {
     group: "network",
   },
 
+  listenAddress6: {
+    type: "string",
+    description: "The IPv6 address to listen for p2p UDP and TCP connections",
+    group: "network",
+  },
+
+  port6: {
+    description: "The TCP/UDP port to listen on. The UDP port can be modified by the --discovery-port6 flag.",
```

Comment:
```suggestion
    description: "The TCP/UDP port to listen on. The UDP port can be modified by the --discoveryPort6 flag.",
```

For ipv4 is also references `--discovery-port` instead of `discoveryPort`